scip-java/src/main/scala/com/sourcegraph/scip_java/buildtools/SbtBuildTool.scala

```diff
@@ -43,8 +43,14 @@ class SbtBuildTool(index: IndexCommand) extends BuildTool("sbt", index) {
 
   private def unconditionallyGenerateScip(): Int =
     Using.resource(sourcegraphSbtPluginFile()) { _ =>
+      val buildCommand =
+        if (index.buildCommand.isEmpty)
+          List("sourcegraphScip")
```

Comment:
@olafurpg what's the approach to custom build commands? 

`sourcegraphScip` only produces the SCIP file itself, do we leave it as the last command anyways, or expect the user to add it in if they need the SCIP index as well?

Reply:
To mirror the Maven/Gradle behavior, the user should only provide commands to compile the codebase to generate SemanticDB